ci/jenkins/templates/conformance-template.yaml

```diff
@@ -25,8 +24,13 @@
                 url: 'https://github.com/{repo-owner}/{repo-name}.git'
                 credentials-id: '{repo-credentials}'
                 branches:
-                    - '{branch}'
+                    - $branch
```

Comment:
Shouldn't this be `${BRANCH}'?

Reply:
if we use `${BRANCH}` then jjb tries to interpolate it and fails due to not finding the branch value anywhere. Using simply `$branch` make jjb ignore it and jenkins use the proper `BRANCH` parameter that we define above :D

Reply:
IIRC for jjb you have to use double brackets on job templates if you want to embed bash variables 
https://docs.openstack.org/infra/jenkins-job-builder/definition.html#macro-notes
Also, I think that is recommended to default the variable always
`${{BRANCH:-}}`

Reply:
nice, thanks for linking to the proper docs, I had a hard time finding that.